governance/src/agent.js

```diff
@@ -0,0 +1,301 @@
+    severity: 'Info',
+    protocol: config.protocolName,
+    metadata: {
+      address,
+      id: proposalId,
+      state: 'executed',
+    },
+  });
+}
+
+function proposalQueuedFinding(proposalId, address, config, eta) {
+  return Finding.fromObject({
+    name: `${config.protocolName} Governance Proposal Queued`,
+    description: `Governance proposal ${proposalId} has been queued`,
+    alertId: `${config.developerAbbreviation}-${config.protocolAbbreviation}-PROPOSAL-QUEUED`,
+    type: 'Info',
+    severity: 'Info',
+    protocol: config.protocolName,
+    metadata: {
+      address,
+      id: proposalId,
+      state: 'queued',
+    },
+  });
+}
+
+function quorumNumeratorUpdatedFinding(address, config, oldNum, newNum) {
+  return Finding.fromObject({
+    name: `${config.protocolName} Governance Quorum Numerator Updated`,
+    description: `Quorum numerator updated from ${oldNum} to ${newNum}`,
+    alertId: `${config.developerAbbreviation}-${config.protocolAbbreviation}-GOVERNANCE-QUORUM-NUMERATOR-UPDATED`,
+    type: 'Info',
+    severity: 'Info',
+    protocol: config.protocolName,
+    metadata: {
+      address,
+      oldNumerator: oldNum,
+      newNumerator: newNum,
+    },
+  });
+}
+
+function timelockChangeFinding(address, config, oldAddress, newAddress) {
+  return Finding.fromObject({
+    name: `${config.protocolName} Governance Timelock Address Change`,
+    description: `Timelock address changed from ${oldAddress} to ${newAddress}`,
+    alertId: `${config.developerAbbreviation}-${config.protocolAbbreviation}-GOVERNANCE-TIMELOCK-ADDRESS-CHANGED`,
+    type: 'Info',
+    severity: 'Info',
+    protocol: config.protocolName,
+    metadata: {
+      address,
+      oldTimelockAddress: oldAddress,
+      newTimelockAddress: newAddress,
+    },
+  });
+}
+
+function votingDelaySetFinding(address, config, oldDelay, newDelay) {
+  return Finding.fromObject({
+    name: `${config.protocolName} Governance Voting Delay Set`,
+    description: `Voting delay change from ${oldDelay} to ${newDelay}`,
+    alertId: `${config.developerAbbreviation}-${config.protocolAbbreviation}-GOVERNANCE-VOTING-DELAY-SET`,
+    type: 'Info',
+    severity: 'Info',
+    protocol: config.protocolName,
+    metadata: {
+      address,
+      oldVotingDelay: oldDelay,
+      newVotingDelay: newDelay,
+    },
+  });
+}
+
+function votingPeriodSetFinding(address, config, oldPeriod, newPeriod) {
+  return Finding.fromObject({
+    name: `${config.protocolName} Governance Voting Period Set`,
+    description: `Voting period change from ${oldPeriod} to ${newPeriod}`,
+    alertId: `${config.developerAbbreviation}-${config.protocolAbbreviation}-GOVERNANCE-VOTING-PERIOD-SET`,
+    type: 'Info',
+    severity: 'Info',
+    protocol: config.protocolName,
+    metadata: {
+      address,
+      oldVotingPeriod: oldPeriod,
+      newVotingPeriod: newPeriod,
+    },
+  });
+}
+
+function proposalThresholdSetFinding(address, config, oldThresh, newThresh) {
```

Comment:
this function was never used - you could add a case statement on line 261-284 where you are filtering for log events and create a new case/alert for this